xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -52,6 +52,29 @@ The next example shows the XAML code that is associated with the previous exampl
     </remarks>
   </Docs>
   <Members>
+    <MemberGroup MemberName="CreateSafeFileHandle">
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.WindowsRuntime</AssemblyName>
+        <AssemblyVersion>4.0.14.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Docs>
+        <summary>Creates and returns a reference to a safe file handle for the IStorage instance that is being extended.</summary>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+With the Windows 10 Anniversary Update, new interfaces were added to <xref:Windows.Storage.IStorageFolder> and <xref:Windows.Storage.IStorageFile> that allow creating a standard Win32 file handle: `IStorageFolderHandleAccess` and `IStorageItemHandleAccess`. The `CreateSafeFileHandle` group of extension methods take advantage of the `Create` methods of these interfaces.
+
+Getting the Win32 handle allows richer interoperability with both .NET and other existing libraries. You can, for example, go straight to a <xref:System.IO.FileStream> without complicated and potentially performance-sapping wrappers.
```

Comment:
These sentences aren't really true. The key here is FileStream access to all files. We removed all of the "expensive" wrapping which is why access is problem. With the wrappers there was also a threading issue, but that is gone too. I'd stay away from this whole statement.

Reply:
Thank you for noticing. I'll remove the two sentences.